provider.tf.example

```diff
@@ -1,18 +1,20 @@
 provider "oci" {
-  fingerprint         = var.api_fingerprint
-  private_key         = var.api_private_key
-  region              = var.region
-  tenancy_ocid        = local.tenancy_id
-  user_ocid           = local.user_id
-  config_file_profile = var.config_file_profile
+  config_file_profile  = var.config_file_profile
```

Comment:
`config_file_profile` should be optional

Reply:
It is optional - the default profile will be used when the var is unset, same as if not included in the provider block.